src/components/reservation/forms/ModalReservationFormSelect.tsx

```diff
@@ -22,9 +23,9 @@ const modalProps = (
 ) => ({
   modalId: modalReservationFormId(type),
   screenReaderModalDescriptionText: t(
-    `screenReaderModalDescription${type.toUpperCase()}Text`
+    `screenReaderModalDescription${upperFirst(type)}Text`
```

Comment:
See comment about regarding dynamically generated translation strings. This case and the rest of this PR is not so bad though.